java/src/Ice/src/main/java/com/zeroc/Ice/TwowayOnlyException.java

```diff
@@ -45,7 +22,7 @@ public String ice_id() {
   }
 
   /** The name of the operation that was invoked. */
-  public String operation;
+  public final String operation;
```

Comment:
I would drop this field, just like I did in C#.

Overall, local exceptions should have very few fields.